plugin.rb

```diff
@@ -177,9 +177,10 @@ class ::User
       WHERE X.id = question_answer_comments.id
     ) Y ON true
     WHERE question_answer_comments.post_id IN (#{post_ids_sql})
+    ORDER BY question_answer_comments.id ASC
     SQL
 
-    QuestionAnswerComment.where("id IN (#{comment_ids_sql})").each do |qa_comment|
+    QuestionAnswerComment.where("id IN (#{comment_ids_sql})").order(id: :asc).each do |qa_comment|
```

Comment:
Funny this was in https://github.com/discourse/discourse-question-answer/pull/21 and I asked if you needed to order both here and in L180